docs/tf/link_pred.rst

```diff
@@ -120,6 +120,7 @@ Now we can initialize the graph engine in local mode:
     ...    storage_type=PartitionStorageType.memory,
     ...    config_path="",
     ...    stream=False,
+    ...    enable_threadpool=False,
```

Comment:
We need to add it, otherwise there will be a test break: "enable_threadpool is not in Namespaces".

Reply:
We just need to do it in this test case (The same as `config_path` and `stream`), because we manually created a "Namespace" object, in actual use case, we don't need to specify it because we already have a default value.

Reply:
Another reason to use a hydra for configuration..